libImaging/Resample.c

```diff
@@ -107,7 +107,7 @@ ImagingPrecompute(int inSize, int outSize, struct filter *filterp,
     if ( ! kk)
         return 0;
 
-    xbounds = calloc(outSize * 2, sizeof(int));
+    xbounds = malloc(outSize * 2 * sizeof(int));
```

Comment:
It's the calloc/malloc change I don't understand, specifically: "memory can be not initialized only in one place."